test/functional/api/v1/webauthn_verifications_controller_test.rb

```diff
@@ -0,0 +1,85 @@
+require "test_helper"
+
+class Api::V1::WebauthnVerificationsControllerTest < ActionController::TestCase
+  should "route new paths to new controller" do
+    route = { controller: "api/v1/webauthn_verifications", action: "create" }
+    assert_recognizes(route, { path: "/api/v1/webauthn_verification", method: :post })
+  end
+
+  def authorize_with(str)
+    @request.env["HTTP_AUTHORIZATION"] = "Basic #{Base64.encode64(str)}"
+  end
+
+  def self.should_respond_to_format(format)
+    context "when the request asks for format '#{format}'" do
+      setup do
+        @user = create(:user)
+        create(:webauthn_credential, user: @user)
+        authorize_with("#{@user.email}:#{@user.password}")
+        post :create, format: format
+      end
+
+      should respond_with :success
+
+      should "return Webauthn verification URL with path token" do
+        assert_not_nil @response.body
+
+        token = @user.webauthn_verification.path_token
+
+        if format == :plain
+          assert_equal @response.body, "example.com/webauthn/#{token}"
+        else
+          response = YAML.load(@response.body)
```

Comment:
In all other test instances I see us using `YAML.safe_load` instead of load directly. Looks like `safe_load` restricts type classes and so would catch any instances in which the returned type was not of an allowed class